README.md

```diff
@@ -5,23 +5,291 @@
+
+```yaml
+profile: sds
+policy:
+  enabled: false
+```
+
+Note that unlike helm, all configurations are validated against a schema, so the operator detects syntax errors. Another
+customization is to define custom namespaces for features:
+
+```yaml
+profile: sds
+
+trafficManagement:
+  namespace: istio-control-custom 
+```
+
+The traffic management feature comprises Pilot, AutoInjection and Proxy components. Each of these components has k8s
+settings, and these can be overridden from the defaults using official k8s APIs (rather than Istio defined schemas):
+
+```yaml
+trafficManagement:
+  components:
+    pilot:
+      k8s:
+        resources:
+          requests:
+            cpu: 1000m # override from default 500m
+            memory: 4096Mi # ... default 2048Mi
+        hpaSpec:
+          maxReplicas: 10 # ... default 5
+          minReplicas: 2  # ... default 1 
+```
+
+The k8s settings are defined in detail in the [operator API](https://github.com/istio/operator/blob/95e89c4fb838b0b374f70d7c5814329e25a64819/pkg/apis/istio/v1alpha1/istioinstaller_types.proto#L394). The settings are the same for all components, so a user can configure pilot k8s settings in exactly the same, consistent way as galley settings. Supported k8s settings currently include:
+
++   [resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container)
++   [readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)
++   [replica count](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
++   [HoriizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
++   [PodDisruptionBudget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#how-disruption-budgets-work)
++   [pod annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
++   [service annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
++   [ImagePullPolicy](https://kubernetes.io/docs/concepts/containers/images/)
++   [priority calss name](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass)
++   [node selector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector)
++   [affinity and anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
+
+All of these k8s settings use the k8s API definitions, so k8s documentation can be used for reference. All k8s overlay
+values are also validated in the operator.
+
+#### Customizing the old values.yaml API 
+
+The new platform install API above deals with k8s level settings. The remaining values.yaml parameters deal with Istio
+control plane operation rather than platform installation and for the time being the operator just passes these through
```

Comment:
this sentence would benefit from periods :)

Reply:
Done.